src/interactions/glue_fs.jl

```diff
@@ -0,0 +1,241 @@
+"""
+A glue interaction that will apply to all atom pairs.
+Finnis-Sinclair and similar interactions should sub-type this type.
+This type should be a GeneralInteraction type. But due to the special
+nature of glue interactions and the restriction to pair interactions
+of the GeneralInteraction type, glue interactions are for now a sub-type of 
+SpecificInteraction.
+"""
+abstract type GlueInteraction <: SpecificInteraction end
+
+"""
+    FinnisSinclairInteraction(nl_only,element_pair_map,params)
+
+The Finnis-Sinclair interaction. This interaction expects units to be of 
+these https://lammps.sandia.gov/doc/units.html units (eV, Å, K, ps and so on).
+"""
+struct FinnisSinclair <: GlueInteraction
+    nl_only::Bool
+    element_pair_map::Dict
+    params::DataFrame
```

Comment:
These are not a concrete types and will cause performance problems. I would recommend using a (parametrized) `NamedTuple` or another (concretely typed) `struct` instead. See https://docs.julialang.org/en/v1/manual/performance-tips/#Avoid-fields-with-abstract-type